src/System-VMEvents/InputEventSensor.class.st

```diff
@@ -576,6 +594,6 @@ InputEventSensor >> yellowButtonPressed [
 	"Answer whether only the yellow mouse button is being pressed. 
 	This is the second mouse button or option+click on the Mac."
 
-	^(self mouseButtons bitAnd: 7) = 2
```

Comment:
Here you can see the other inconsistency with the original shared pool class variable value.